CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 18.2.0 (TODO)
+
+### React
+
+* Only treat updates to Suspended components as a new mount in Legacy Mode ([@rickhanlonii](https://github.com/rickhanlonii)) in [#24530](https://github.com/facebook/react/pull/24530)
+
+### React DOM
+
+* Provide a Component Stack in Dev to `onRecoverableError` through an `errorInfo` second argument, similar to `componentDidCatch` in Error Boundaries ([@gnoff](https://github.com/gnoff)) in [#24591](https://github.com/facebook/react/pull/24591)
+* Support `hydrateRoot` hydrating into `Document` as the `container` ([@gnoff](https://github.com/gnoff)) in [#24523](https://github.com/facebook/react/pull/24523)
```

Comment:
This was already supported, the bug was that it couldn't cleanly client-render into it. I'd maybe rephrase as "Fix hydrating into document causing a blank page on mismatch".